addons/edi/views/edi_document_type_views.xml

```diff
@@ -1,90 +1,90 @@
 <?xml version="1.0"?>
 <odoo>
-  <data>
+	<data>
 
-    <!-- Form view -->
-    <record id="document_type_form" model="ir.ui.view">
-      <field name="name">edi.document.type.form</field>
-      <field name="model">edi.document.type</field>
-      <field name="arch" type="xml">
-	<form string="EDI Document Type">
-	  <sheet>
-	    <div class="oe_title">
-	      <label for="name" class="oe_edit_only"/>
-	      <h1>
-		<field name="name" placeholder="e.g. Product Master Data"/>
-	      </h1>
-	    </div>
-	    <group>
-	      <group name="basic">
-		<field name="model_id"/>
-		<field name="rec_type_ids" widget="many2many_tags"/>
-		<field name="sequence_id"/>
-		<field name="processing_level"/>
-		<field name="execute_sequentially"/>
-	      </group>
-	      <group name="issues" string="Issues">
-		<field name="project_id"/>
-		<field name="fail_fast"/>
-	      </group>
-	      <group name="extras" string="Extras"/>
-	    </group>
-	  </sheet>
-	</form>
-      </field>
-    </record>
+		<!-- Form view -->
```

Comment:
what's up with the indentation in this file?

Reply:
![image](https://github.com/user-attachments/assets/a99a1f90-a11a-44b8-be65-bdbca12749fb)
They have different indentation used, space and tab mixed i think. It looks weird on pycharm